docs/releasing/releasing.md

```diff
@@ -0,0 +1,96 @@
+
+The following custom actions support the release pipeline:
+
+- [release-please.yml](https://github.com/trussworks/react-uswds/actions/workflows/release-please.yml) enables [release-please](#release-please) as described above.
+- [package-release.yml](https://github.com/trussworks/react-uswds/actions/workflows/package-release.yml) packages a clean, publishable build artifact for each release commit.
+
+### [npm](https://www.npmjs.com/package/@trussworks/react-uswds?activeTab=versions)
+
+Publish and host versioned build artifacts for the community to download as a project dependency.
+
+## Release and Publish React USWDS
+
+### Prerequisites
+
+To performa a release, a maintainer must first
+
+- Be part of the [React USWDS Admins team](https://github.com/orgs/trussworks/teams/react-uswds-admins).
+- Be an Admin or Owner of the [trussworks npm organization](https://www.npmjs.com/settings/trussworks/members).
+
+Additionally, at least one releasable change must have been merged to main since the previous release.
+
+> [!NOTE]
+> If there are no releasable changes as defined by the [release-please configuration](https://github.com/trussworks/react-uswds/blob/main/release-please-config.json), there will be no Release PR to start the release process with.
+
+### Step-by-step
+
+Follow these steps to create and publish a release for a given version, `<major.minor.incremental>`.
+
+1. Review and merge the current [release PR](https://github.com/trussworks/react-uswds/issues?q=is%3Apr%20author%3Aapp%2Fgithub-actions%20label%3A%22type%3A%20release%22).
+   - Ensure the branch is up to date with `main`.
+   - GitHub actions cannot initiate other GitHub action workflow runs, so CI checks on this PR will not start if `release-please` updated the branch last. In this case, bypass merge restrictions as an administrator to merge.
+2. Wait for `release-please` to create the corresponding [tag](https://github.com/trussworks/react-uswds/tags) in GitHub including the updates from the changelog.
+3. Find and wait for the corresponding [package-release.yml](https://github.com/trussworks/react-uswds/actions/workflows/package-release.yml) action to complete.
+4. From the successful package-release workflow run, download the build artifact from the `Artifacts` section of the build summary:
+   ![Where to find the build artifact for download](package-release-summary-artifact-download.png)
+5. Get the tarball (`trussworks-react-uswds-v<major.minor.incremental>.tgz`) by unzipping the downloaded `artifact.zip`
+
+   ```shell
+   unzip artifact.zip
+   ```
+
+   - If, instead of the terminal, you wish to instead use Archive Utility to unzip `arifact.zip` be aware that Archive Utility defaults to recursively unzipping files, including the `.tgz` itself `.tgz` file when unzipping the `artifact.zip`. You will first need to disable the "Keep expanding if possible" setting in your Archive Utility preferences.
```

Comment:
```suggestion
   - If, instead of the terminal, you wish to use Archive Utility to unzip `arifact.zip` be aware that Archive Utility defaults to recursively unzipping files, including the `.tgz` itself `.tgz` file when unzipping the `artifact.zip`. You will first need to disable the "Keep expanding if possible" setting in your Archive Utility preferences.
```